keras_eval/eval.py

```diff
@@ -145,6 +149,32 @@ def evaluate(self, data_dir=None, top_k=1, filter_indices=None, confusion_matrix
 
         return self.probabilities, self.labels
 
+    def evaluate_combination_concepts(self):
+        '''
+        Sets probabilities if model concepts are to be combined in test
+        '''
+
+        if utils.compare_group_test_concepts(self.concept_labels, self.concept_dictionary) and utils.check_concept_unique(self.concept_dictionary):
```

Comment:
Why not combining `utils.compare_group_test_concepts` and `utils.check_concept_unique` to be a single function `utils.check_concept_dictionary`? 

I don't see other uses `compare_group_test_concepts` and `check_concept_unique` can have individually.